source/extensions/filters/http/istio_stats/config.proto

```diff
@@ -111,4 +111,10 @@ message PluginConfig {
 
   // Proxy deployment type.
   Reporter reporter = 10;
+
+  // Metric expiry duration. Defaults to 12h. Set to 0 to disable the metric expiry.
+  google.protobuf.Duration expiry_duration = 11;
```

Comment:
- Upon expiration, what happens for a metric?
- If expiry_duration is used to assign the value of rotate_interval_ms, should expiry_duration be called rotate_interval?

Reply:
I'm thinking make this default to `0` in the first release, so that the behavior is same as prev  release.
We can change the default value later, WDYT?

Reply:
@lei-tang I'll add more code comments to explain how this works.
@zirain That would be fine.